.github/workflows/tests.yml

```diff
@@ -18,7 +18,7 @@ concurrency:
 
 jobs:
   unit-tests:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-latest
```

Comment:
We are just discussing this on other PR. But I don't think it should block this PR.

Reply:
OK let's merge this. But please take care of keeping such changes split up, not hidden inside random commits @adamkankovsky